pkg/registry/ociregistry_test.go

```diff
@@ -0,0 +1,183 @@
+package registry
```

Comment:
could we move all tests to `registry_test`

Reply:
Done.